.github/workflows/generate-dependency-workflows.yml

```diff
@@ -63,24 +68,23 @@ jobs:
             > ".github/workflows/${name}-test-upload-metadata.yml"
         done
 
-    - name: Commit workflows
+    - name: Commit
       id: commit
-      run: |
-        git add -AN .github/workflows
-        if ! git diff --quiet .github/workflows; then
-          git config user.name "Core Deps"
-          git config user.email coredependencies@vmware.com
-          git add .github/workflows/*-get-new-versions.yml
-          git add .github/workflows/*-build-upload.yml
-          git add .github/workflows/*-test-upload-metadata.yml
-          git commit -m 'Update dependency workflows'
+      uses: paketo-buildpacks/github-config/actions/pull-request/create-commit@main
+      with:
+        message: "Update dependency workflows"
+        pathspec: ".github/workflows/*-get-new-versions.yml  .github/workflows/*-build-upload.yml .github/workflows/*-test-upload-metadata.yml"
```

Comment:
Have y'all seen this work in a github workflow? Based on [this implementation](https://github.com/paketo-buildpacks/github-config/blob/2d8aaa29b5709a390bfbe6b647d2472555b4883b/actions/pull-request/create-commit/entrypoint#L54) I suspect it would look for a single file with the name `.github/workflows/*-get-new-versions.yml  .github/workflows/*-build-upload.yml .github/workflows/*-test-upload-metadata.yml`.